app/styles/navbar.css

```diff
@@ -23,15 +23,15 @@ nav li {
 }
 
 nav li a:hover {
-  color: #49a82e;
+  color: var(--nav--link--hover);
 }
 
 .nav-menu {
   list-style-type: none;
   margin: 0;
   padding: 0;
   overflow: hidden;
-  background-color: #1d1283;
+  background-color: var(--nav-menu--bg);

```

Comment:
On line 9, this color is already used as navbar background, can we please reuse it ?